locales/en/yourloops.json

```diff
@@ -513,6 +516,7 @@
   "time-in-range": "TIR",
   "time-in-range-cgm-daily-average": "**Time In Range:** Daily average of the time spent in range, based on CGM readings.",
   "time-in-range-cgm-one-day": "**Time In Range:** Time spent in range, based on CGM readings.",
+  "time-in-range-tooltip": "Time in Range",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** The acronym needs to be respected‏
```suggestion
  "time-in-range-tooltip": "Time In Range",
```